app/Services/MetadataService.php

```diff
@@ -34,13 +33,19 @@ public function __construct()
     public function index($metaData)
     {
         try {
-            $params       = $this->getIndexType();
-            $params['id'] = $metaData['id'];
-            $document     = $this->es->exists($params);
-            $metadata     = json_decode($metaData['metadata']);
-            $createdBy    = json_decode($metaData['created_by']);
-            $updatedBy    = json_decode($metaData['updated_by']);
-            $data         = [
+            $params                 = $this->getIndexType();
+            $params['id']           = $metaData['id'];
+            $document               = $this->es->exists($params);
+            $metadata               = json_decode($metaData['metadata']);
```

Comment:
is it multi-dimension json data ? if it is; then set second flag true for associative array output json_decode($metaData['metadata'],true);

Reply:
It will give us object instead of array